provide installation instructions

from readme: `run make`

there are no installation instructions and it seems that installation instructions are necessary, as simply running make for me results in:

```
mateusz@grisznak:~/Desktop/tmp/name-suggestion-index$ make
module.js:328
    throw err;
    ^

Error: Cannot find module 'json-stable-stringify'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/mateusz/Desktop/tmp/name-suggestion-index/build.js:6:17)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
Makefile:2: recipe for target 'all' failed
make: *** [all] Error 1
```

I guessed and expected `npm install` to work but it also failed.

--- a/README.md
+++ b/README.md
@@ -44,15 +44,18 @@ completely ignored
 - `name-suggestions.json` and `name-suggestions.min.json` are compiled, any changes made to them 
 directly will be overwritten
 
-### Updating topNames.json from planet
+### Installation
 - `git clone https://github.com/osmlab/name-suggestion-index.git && cd name-suggestion-index`
-- [download the planet](http://planet.osm.org/pbf/)
-    - `wget http://planet.osm.org/pbf/planet-latest.osm.pbf`
 - Ubuntu install (install recent nodejs):
     - `sudo apt-get update`
     - `sudo apt-get install -y python-software-properties python g++ make`
     - `sudo apt-get install nodejs npm`
     - `sudo ln -s /usr/bin/nodejs /usr/bin/node`
     - `npm install`
+
+### Updating topNames.json from planet
+- install as described above
+- [download the planet](http://planet.osm.org/pbf/)
+    - `wget http://planet.osm.org/pbf/planet-latest.osm.pbf`
 - `node getRaw yourOSMfile`
     - results will go in `topNames.json`

